framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -76,15 +76,17 @@
  *       error if any are found.
  * </ol>
  *
- * <p>Steps 3 and 4 are separated so that an error is issued only once per invalid expression string
+ * <p>Steps 2 and 3 are separated so that an error is issued only once per invalid expression string
  * rather than every time the expression string is parsed. (The expression string is parsed multiple
  * times because annotated types are created multiple times.)
  */
 public class DependentTypesHelper {
+
+    /** AnnotatedTypeFactory */
     protected final AnnotatedTypeFactory factory;
 
     /** A map of annotation classes to the names of their elements that are Java expressions. */
```

Comment:
I think this should map to a list of ExecutableElement, not String.  That can be more efficient.